tabled_derive/src/parse/type_attr.rs

```diff
@@ -14,6 +14,7 @@ pub fn parse_type_attributes(
 }
 
 pub struct TypeAttr {
+    #[allow(dead_code)]
```

Comment:
shouldn't we also remove this `dead code`?

Reply:
Damn, missed it, thanks

Reply:
that's what `reviews` are for